### PR TITLE
Gdpr add

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
pls review changes

### CODE REVIEW

File: src/components/Sidebar/Untitled-1.txt [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
```
Comment (ShankyRana) [inline src/components/Sidebar/Untitled-1.txt]: what is this file?

File: src/components/WebsitePricing/WebsitePricing.js [hunk 2]
```diff
@@ -59,10 +59,13 @@ class WebsitePricing extends Component {
     return planList.map(plan => {
       return <div className="col-md-3 pl-0 pr-0">
         <div className="pricing pricing-1 boxed boxed--border boxed--lg pr-0 pl-0">
-          <h4>{plan.name}</h4>
+        {(plan.name === "Enterprise Yearly Plan" || plan.name === "Enterprise Monthly Plan") &&  <h3 className="h3 text-center">Enterprise</h3>}
```
Comment (ShankyRana) [inline src/components/WebsitePricing/WebsitePricing.js]: get these into a function with switch case

File: src/components/WebsiteHome/WebsiteHome.js [hunk 3]
```diff
@@ -4,38 +4,39 @@ import Slider from 'react-slick';
...
-  HigherConversion2,
-  Saveonacquisitions2,
-  Visitortrust2,
-  Startconverting,
-  Socialinfluence2,
-  D12, B24,
-  Customerjourney5,
-  Analyticsinsights10,
-  Easiestsetup,
+  // HigherConversion,
```
Comment (ShankyRana) [inline src/components/WebsiteHome/WebsiteHome.js]: if these are not required, remove them

Comment (GAUTAMRAJU15) [reply src/components/WebsiteHome/WebsiteHome.js]: yes not required i will remove it

File: src/components/WebsitePricing/WebsitePricing.js (hunk 2 above)
Comment (GAUTAMRAJU15) [reply src/components/WebsitePricing/WebsitePricing.js]: ok sir

**Pull request closed without merging**